<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CORAL Browser</string>
  </property>
  <property name="styleSheet">
   <string notr="true">padding: {0px}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Tab"/>
    <addaction name="actionNew_Window"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionShow_Spent_Time"/>
   </widget>
   <widget class="QMenu" name="connectionIcon">
    <property name="title">
     <string>&quot;</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-CORALBrowser-MSVC_32Bit-Debug/debug/res/images/Offline.png</normaloff>../build-CORALBrowser-MSVC_32Bit-Debug/debug/res/images/Offline.png</iconset>
    </property>
    <addaction name="actionLog_Out"/>
   </widget>
   <addaction name="connectionIcon"/>
   <addaction name="menuFile"/>
   <addaction name="menuDebug"/>
  </widget>
  <action name="actionNew_Tab">
   <property name="text">
    <string>New Tab</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionShow_Spent_Time">
   <property name="text">
    <string>Show Spent Time</string>
   </property>
  </action>
  <action name="actionLog_Out">
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
